catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -64,6 +64,7 @@
         <bundle>mvn:ddf.catalog.core/catalog-core-resourcesizeplugin/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-camelcomponent/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-dynamic-registry/${project.version}</bundle>
+        <bundle>mvn:ddf.catalog.core/catalog-core-migratable/${project.version}</bundle>
```

Comment:
Is this to be an irremovable component of `catalog-core` or should it be wrapped in its own installable feature with a pre-req on `catalog-core`?